app/components-react/pages/stream-scheduler/useStreamScheduler.tsx

```diff
@@ -217,9 +217,9 @@ class StreamSchedulerModule {
    * Shows a modal for creating a new event
    */
   @mutation()
-  showNewEventModal(platform: TPlatform, selectedTime?: Moment) {
+  showNewEventModal(platform: TPlatform, selectedTime?: number) {
```

Comment:
Ah yes all args of mutations need to be serializable otherwise there will be issues.

Reply:
Ah wait this is a redux module not a vuex module.  They look so similar :)